### PR TITLE
termshark: add version 2.2.0

### PR DESCRIPTION
**Pull request by Slach**
Hello
`termshark` is a terminal UI for tshark, inspired by Wireshark

### CODE REVIEW

File: bucket/termshark.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "2.2.0",
+    "description": "A terminal UI for tshark, inspired by Wireshark",
+    "homepage": "https://github.com/gcla/termshark",
+    "license": "MIT",
+    "depends": "wireshark",
+    "notes": [
+        "Run 'scoop install wireshark' (from a elevated shell) to install 'tshark.exe' automatically"
+    ],
+
```
Comment (Ash258) [inline bucket/termshark.json:10]: So lets depend on manifest and then say to users this:

> Hey I just installed term shark and wireshark as dependency, but execute install command for wireshark manually to see a error

File: bucket/termshark.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/gcla/termshark/releases/download/v$version/termshark_$version_windows_x64.zip",
+                "hash": {
+                    "url": "https://github.com/gcla/termshark/releases/download/v$version/checksums.txt",
+                    "regex": "$sha256\\s+$basename"
+                }
+            }
+        }
```
Comment (Ash258) [inline bucket/termshark.json:33]:
```suggestion
                "url": "https://github.com/gcla/termshark/releases/download/v$version/termshark_$version_windows_x64.zip",
                "extract_dir": "termshark_$version_windows_x64"
            }
        },
        "hash": {
            "url": "$baseurl/checksums.txt",
            "regex": "$sha256\\s+$basename"
        }
```

File: bucket/termshark.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "license": "MIT",
+    "depends": "wireshark",
+    "notes": [
+        "Run 'scoop install wireshark' (from a elevated shell) to install 'tshark.exe' automatically"
+    ],
+
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/gcla/termshark/releases/download/v2.2.0/termshark_2.2.0_windows_x64.zip",
+            "hash": "63e64e2d86d36fe73d2a32ebada46d1a8445e44ab7d4e01253053dbfbd73cc12"
```
Comment (Ash258) [inline bucket/termshark.json:14]:
```suggestion
            "hash": "63e64e2d86d36fe73d2a32ebada46d1a8445e44ab7d4e01253053dbfbd73cc12",
            "extract_dir": "termshark_2.2.0_windows_x64"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Do not work

image

File: bucket/termshark.json (hunk 1 above)
Comment (Slach) [reply bucket/termshark.json:10]: @Ash258 could you suggest a better solution? 

sorry for late response

**Comment by Slach** [comment]
@Ash258
I applied your suggestions, thanks a lot
sorry for late response

but after `scoop install termshark.json` I see only `termshark.lnk` but not see `scoop\shim\ternshark.exe` could you suggest how to fix it?

**Comment by Ash258** [comment]
Why would you create dumb shim for appllication which does not have command line support???

**Comment by rashil2000** [comment]
Hi @Slach, can you move this PR to Main bucket? Thanks.

**Comment by Slach** [comment]
@rashil2000 done, I close this PR

**Pull request closed without merging**